src/styles/main.scss

```diff
@@ -0,0 +1,8 @@
+@import "./utils/variables.scss";
+@import "./utils/functions.scss";
+@import "./utils/mixins.scss";
```

Comment:
Consider renaming `functions.scss` to `_functions.scss` to follow the partial naming convention in SCSS.